packages/engine/Source/Scene/Cesium3DTileset.js

```diff
@@ -1545,17 +1571,123 @@ Object.defineProperties(Cesium3DTileset.prototype, {
    *
    * @exception {DeveloperError} <code>maximumMemoryUsage</code> must be greater than or equal to zero.
    * @see Cesium3DTileset#totalMemoryUsageInBytes
+   *
+   * @deprecated
    */
   maximumMemoryUsage: {
     get: function () {
+      deprecationWarning(
+        "Cesium3DTileset.maximumMemoryUsage",
+        "Cesium3DTileset.maximumMemoryUsage was deprecated in CesiumJS 1.106.  It will be removed in CesiumJS 1.109. Use Cesium3DTileset.cacheBytes instead."
+      );
       return this._maximumMemoryUsage;
     },
     set: function (value) {
+      deprecationWarning(
+        "Cesium3DTileset.maximumMemoryUsage",
+        "Cesium3DTileset.maximumMemoryUsage was deprecated in CesiumJS 1.106.  It will be removed in CesiumJS 1.109. Use Cesium3DTileset.cacheBytes instead."
+      );
       //>>includeStart('debug', pragmas.debug);
       Check.typeOf.number.greaterThanOrEquals("value", value, 0);
       //>>includeEnd('debug');
 
       this._maximumMemoryUsage = value;
+      this._cacheBytes = value * 1024 * 1024;
+    },
+  },
+
+  /**
+   * The amount of GPU memory (in bytes) used to cache tiles. This memory usage is estimated from
+   * geometry, textures, and batch table textures of loaded tiles. For point clouds, this value also
+   * includes per-point metadata.
+   * <p>
+   * Tiles not in view are unloaded to enforce this.
+   * </p>
+   * <p>
+   * If decreasing this value results in unloading tiles, the tiles are unloaded the next frame.
+   * </p>
+   * <p>
+   * If tiles sized more than <code>cacheBytes</code> are needed to meet the
+   * desired screen space error, determined by {@link Cesium3DTileset#maximumScreenSpaceError},
+   * for the current view, then the memory usage of the tiles loaded will exceed
+   * <code>cacheBytes</code>.  For example, if the maximum is 256 MB, but 300 MB
+   * of tiles are needed to meet the screen space error, then 300 MB of tiles may
+   * be loaded. When these tiles go out of view, they will be unloaded.
```

Comment:
This description is a bit incomplete without also mentioning `maximumCacheOverflowBytes`